Makefile

```diff
@@ -40,6 +40,9 @@ CXXGEN   := $(BUILDDIR)/cpp
 PYGEN    := $(BUILDDIR)/python
 
 # generated Documentation files
+# default to asciidoc template
+# for mk-docs formatting, pass in TEMPLATE pointing to the mk-docs template
+TEMPLATE := $(SRCDIRINV)/asciidoc.mustache
```

Comment:
no, it should be $(SRCDIRINV)/scripts/asciidoc.mustache

this way, without explicitly setting TEMPLATE make will use the default asciidoc.mustache template in the mk-protobuf repo

during docs build, TEMPLATE is overridden at the command line

will fix